community/portworx/values.yaml

```diff
@@ -37,7 +37,7 @@ internalKVDB: false                   # internal KVDB
 
 etcd:
   secret: none                       # Secret name where the username, password and CA cert for ETCD authentication is stored
-imageVersion: 2.3.2                   # Version of the PX Image.
+imageVersion: 2.3.5                   # Version of the PX Image.
```

Comment:
Should we update this to 2.4.0 if its released, before we raise this PR to the main repo?